SPECS/privacyidea-radius.spec

```diff
@@ -54,6 +54,12 @@ cp $RPM_SOURCE_DIR/privacyidea-mods-perl $RPM_BUILD_ROOT/etc/raddb/mods-availabl
 rm $RPM_BUILD_ROOT/git -fr
 
 
+%post
+# Activate the piperl RADIUS module
+cd /etc/raddb/mods-enabled/
+ln -s ../mods-available/piperl .
+systemctl restart radiusd
```

Comment:
I am not sure, if we also should activate the sites-available/privacyidea.
But on the other hand, we then would also need to deactivate other site definitions.

In contrast to this, the piperl is a privacyidea specific module, which will not hurt anyone.

So I think it might be better, to *not* activate the sites-available/privacyidea automatically, if s.o. choose to run another config.

Reply:
sounds Ok as long as we have that info written down somewhere.
By default freeradius will be disabled and inactive after the installation in CentOS. The ``restart`` will start the daemon but _not_ enable it, so at the next reboot it won't come up. I have to look up what the policy is in other CentOS packages.

Reply:
The more I am thinking about this, I think we should take care to not break anything and let the user decide:

* do not activate radius, do not restart radius and do not disable other sites in radius.
* Rather add a note to the info we send, when the user gets the download information:

~~~~
* enable the default privacyidea radius definition by
  ln -s /etc/raddb/sites-available/privacyidea /etc/raddb/sites-enabled/
* please take care do disable radius sites at /etc/raddb/sites-enabled/
  we recommend deleting all links except the privacyidea - unless you know what you are doing
* please take care to activate the freeradius at boot time by issuing
  systemctl enable radiusd
~~~~
Everyone, who installs from CentOS7 repos will get this information (since it also contains the link to the repo). 
There is no sensible way to add this in the RPM itself.
So I would simply merge it that way.

Reply:
> So I would simply merge it that way.

So we remove the restart of radiusd?

Reply:
And we can write ``systemctl enable radiusd --now`` which enables _and_ starts the radius daemon.

Reply:
I think we confirmed that we do not enable freeradius, right.